<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/newPrefix/wrench-watch-dogs-2-monochorme_1565054351.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Sitka Text&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);



</string>
    </property>
    <property name="text">
     <string>Choose the Path to the Video:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>991</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>680</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pathlabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>911</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Sitka Text&quot;;
background-color: rgb(12, 12, 12);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_trackingtwo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>690</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dimselector">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>Select Dimension</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="folderselect">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 55 22pt &quot;Open Sans Semibold&quot;;
background-color: rgb(53, 53, 53);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>....</string>
    </property>
   </widget>
   <widget class="QPushButton" name="folderselect_csv">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 55 22pt &quot;Open Sans Semibold&quot;;
background-color: rgb(53, 53, 53);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>....</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Pixeled&quot;;
color: rgb(255, 255, 255);



</string>
    </property>
    <property name="text">
     <string>Choose the Path to save CSV:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twodim_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>991</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);
font: 63 18pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(95, 95, 95);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pathlabel_csv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>911</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Sitka Text&quot;;
background-color: rgb(12, 12, 12);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>twodim_5</zorder>
   <zorder>folderselect_csv</zorder>
   <zorder>pathlabel_csv</zorder>
   <zorder>label_2</zorder>
   <zorder>twodim_3</zorder>
   <zorder>error</zorder>
   <zorder>label</zorder>
   <zorder>twodim_2</zorder>
   <zorder>twodim_4</zorder>
   <zorder>pathlabel</zorder>
   <zorder>cancel_trackingtwo</zorder>
   <zorder>dimselector</zorder>
   <zorder>folderselect</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="paths elect.qrc"/>
 </resources>
 <connections/>
</ui>
